src/mavlink_receive.c

```diff
@@ -100,6 +100,27 @@ static void mavlink_handle_msg_timesync(const mavlink_message_t *const msg)
   }
 }
 
+static void mavlink_handle_msg_small_imu_hil(const mavlink_message_t * const msg)
```

Comment:
I also didn't wan't it to be confusing, so I figured it made it more clear to have two types